meta/store.go

```diff
@@ -868,6 +868,10 @@ func (s *Store) CreateDatabase(name string) (*DatabaseInfo, error) {
 			return nil, fmt.Errorf("read: %s", err)
 		}
 
+		if nodeN > 3 {
```

Comment:
Ah, OK, cool. 

Not a huge fan of the magic number. Can you lift it up to `MaxAutocreateRPFactor` or something?